assets/css/style.css

```diff
@@ -0,0 +1,268 @@
+:root {
+  --light-gray: #d6d6d6ff;
+  --eerie-black: #222121;
+}
+
+* {
+  border: 0px;
+  margin: 0px;
+  box-sizing: border-box;
+  margin-block-start: 0px;
+  margin-block-end: 0px;
+}
+
+body {
+  height: 100%;
+  width: 100%;
+  background-image: url("../images/html-images/background-image.png");
+  background-size: cover;
+  background-position: center;
+  font-family: "Ubuntu", sans-serif;
+}
+
+/* This is the class selector for class name of header */
```

Comment:
Great to see you including comments but they don't need to be this detailed - you can simply just write `/* header section */` for example

Reply:
Done!!